antarest/study/storage/variantstudy/snapshot_generator.py

```diff
@@ -67,7 +67,7 @@ def __init__(
     def generate_snapshot(
         self,
         variant_study_id: str,
-        jwt_user: JWTUser,
+        owner: JWTUser,
```

Comment:
Why did you change this name ? I think it was better before as it's a JWTUser class. Also I believe any user can reach this code, it doesn't have to be the study owner